docs/core/integration-quality-scale/rules/iqs005.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "IQS005: Support config entry unloading"
+---
+
+IQS005 is as follows:
+
+Support config entry unloading
+
+## Reasoning
+
+Sometimes a config entry ends up in a state where it is stuck.
+This can have many reasons, but sometimes the integration works again after the integration has been reloaded.
+
+In order to support this, the integration should support unloading the config entry.
+This gives users the ability to reload the integration without having to restart Home Assistant.
+Reloading an integration can be a powerful tool for users to fix issues with integrations.
```

Comment:
```suggestion
Sometimes a config entry ends up in a state where it is stuck.
This can be for many reasons, but the integration may work again after it has been reloaded.

In order to support this, the integration should support unloading the config entry.
This gives users the ability to reload the integration without having to restart Home Assistant.
Reloading an integration can be a powerful tool for users to fix issues with integrations.
```